.github/ISSUE_TEMPLATE/release_template.md

```diff
@@ -61,6 +61,7 @@ __REPLACE with OpenSearch wide initiatives to improve quality and consistency.__
 ### Release testing - _Ends __REPLACE_RELEASE-minus-6-days___
 
 - [ ] Declare a release candidate build, and provide the instructions with the release candidates for teams on testing (__REPLACE_RELEASE-minus-8-days__).
+- [ ] Stop builds for this version of OpenSearch and/or OpenSearch Dashboards in order to avoid accidental commits going in unknowingly. Restart only if necessary else manually run the build workflow and declare new release candidate.
```

Comment:
Maybe we can also say enable the docker build with build number tag at this phase?
https://github.com/opensearch-project/opensearch-build/blob/b0930d1a256dd22738e218324cb51a934379fad7/jenkins/opensearch/distribution-build.jenkinsfile#L41-L44

Reply:
I think that is a implementation detail of creating a release candidate and not a process as such.